src/main/java/org/fcrepo/migration/Migrator.java

```diff
@@ -37,27 +38,56 @@ public static void main(String [] args) throws IOException, XMLStreamException {
             Migrator m = new Migrator(new FoxmlDirectoryObjectSource(foxmlRoot, dsRoot, working),
                     new ConsoleLoggingStreamingFedoraObjectHandler());
             m.run();
```

Comment:
Throughout the code, feel free to delete sections rather than commenting them out.